test/Maestro/EU/test.yml

```diff
@@ -0,0 +1,6 @@
+appId: net.givtapp.ios.test
```

Comment:
Do you want to keep this file @donnaluza 
Normally test stuff like this we don't commit into the codebase